src/proto_conversions.cc

```diff
@@ -373,7 +373,9 @@ bool Options_ExpansionProperties_Enum_Parse(const std::string& prop,
               {"durations", Options_ExpansionProperties_durations},
               {"distances", Options_ExpansionProperties_distances},
               {"statuses", Options_ExpansionProperties_statuses},
-              {"edge_ids", Options::ExpansionProperties::Options_ExpansionProperties_edge_ids}};
+              {"edge_ids", Options::ExpansionProperties::Options_ExpansionProperties_edge_ids},
```

Comment:
we might want to rename that? previously it was the same field name in the response, but it's the singular version i.e. `edge_id` instead of `edge_ids`. would be nicer to have them consistent IMO.

Reply:
ok, let me get that in real quick. also the shorter version.